service/alexaforbusiness/api_op_ListSkills.go

```diff
@@ -118,6 +119,85 @@ func addOperationListSkillsMiddlewares(stack *middleware.Stack, options Options)
 	return nil
 }
 
+// ListSkillsAPIClient is a client that implements the ListSkills operation.
+type ListSkillsAPIClient interface {
+	ListSkills(context.Context, *ListSkillsInput, ...func(*Options)) (*ListSkillsOutput, error)
+}
+
+var _ ListSkillsAPIClient = (*Client)(nil)
+
+// ListSkillsPaginatorOptions is the paginator options for ListSkills
+type ListSkillsPaginatorOptions struct {
+	// The maximum number of results to include in the response. If more results exist
+	// than the specified MaxResults value, a token is included in the response so that
+	// the remaining results can be retrieved.
+	Limit *int32
```

Comment:
Even though this member is modeled as boxed, should the paginator `Limit` option also treat it as boxed? Or would it be better to force the `Limit` member to value types, if zero do nothing.

A boxed `Limit` member suggests the user could request zero length pages which makes no sense.